### DISCUSSION TITLE
Docker image for eXo community.

### DISCUSSION THREAD

**Discussion by mvsingh007** [exoplatform/exo-community#29] [Q&A]
Hello Sir / Madam,

I need to download docker image for eXo community for ubuntu 22.4.  from where  I can download the same , required URL.

thanks & regards,

Manvir Singh

**Comment by rdenarie** [comment]
Hello @mvsingh007 
To start eXo Community, you can follow our documentation : 
https://docs.exoplatform.org/guide/getting-started/start-community.html

In this, you will find the dockerfile which allow to start all the required containers.

**Comment by mvsingh007** [comment]
Hello Sir / Madam,
Greetings…. How are you ….?
Thanks for the reply.. Can you please confirm the base operation system ( Version ) on which docker will places.

Best Regards
Manvir Singh
Motherson Technology Services Limited (MTSL)
(Formerly MothersonSumi INfotech & Designs Limited).

**Reply by rdenarie** [reply]
Hello,
Containers use Ubuntu 22.04 as operating system.

**Comment by mvsingh007** [comment]
Thank you so much for your support. It is installed successfully.
Now question is, how we can take backup for our changes.

Best Regards
Manvir Singh
Motherson Technology Services Limited (MTSL)
(Formerly MothersonSumi INfotech & Designs Limited).

**Reply by rdenarie** [reply]
If you look at docker-compose file, each component have a "volumes" part. 
Example for exo component : 

>      volumes:
>       - exo_data:/srv/exo
>       - exo_logs:/var/log/exo
> 

Docker will create a volume to store exoplatform data, and a volume for logs. By default, volumes are located in /var/lib/docker/volumes, but you can move it where you want in the host.
If you want to backport the platform, you need to backport all the volumes availables in docker_compose
